composer.json

```diff
@@ -24,6 +24,9 @@
   "require": {
     "php": ">=5.3.0"
   },
+  "require-dev": {
+    "phpstan/phpstan": "^0.12.58"
+  },
```

Comment:
This will make the project not composer compatible with PHP 5 @nicolaasuni ⚠️

Reply:
done